admin/workspace-management/self-contained-builds.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Self-contained workspace builds"
+description: Learn how to enable self-contained workspace builds.
+state: alpha
+---
+
+By default the Coder workspace boot sequence occurs remotely -- Coder uploads
+assets (including the Coder agent, code-server, and JetBrains Projector) from
+`coderd` to a workspace.
+
+However, Coder offers the option of using **self-contained workspace builds**.
+Enabling this option changes the Coder deployment so that workspaces control the
+boot sequence internally, with Coder downloading assets from `coderd` to the
+workspace when the latter starts.
```

Comment:
```suggestion
boot sequence internally, with the workspace downloading assets from `coderd`.
```

This part, `with Coder downloading assets from 'coderd'` is a bit confusing IMO. The workspace downloads the assets from `coderd`.

Trying to make it clear the assets are pulled instead of pushed.